docker/bin/build-docker-images.sh

```diff
@@ -18,7 +18,7 @@ do
                  -t ${DOCKER_REPO}:${image}-${GIT_SHA_SHORT} \
                  --cache-from ${DOCKER_REPO}:${image}-latest \
                  --cache-from kitsune:${image}-latest \
-                 -f docker/dockerfiles/${image} \
```

Comment:
@ziegeer could you please elaborate a bit about this change?

Reply:
Yes, this script is for building docker images locally, which I don't think anyone actually does anymore, but I had to and it pointed to some non-existant dockerfiles that look to have been previously consolidated into a single Dockerfile.  So I just pointed the script at that and it worked.